src/core/event-handle.js

```diff
@@ -121,6 +121,11 @@ class EventHandle {
     get removed() {
         return this._removed;
     }
+
+    // don't stringify EventHandle to JSON by JSON.stringify
+    toJSON(key) {
+        return undefined;
+    }
```

Comment:
Why was this causing a problem? Does this really belong in this PR?

Reply:
I added this to GraphicsDevice back then for the same reason - lots of device properties were serialized with the light.